draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -462,7 +462,7 @@ As an example:
 
     _8443._foo.api.example.com. 7200 IN SVCB 0 svc4.example.net.
     svc4.example.net.  7200  IN SVCB 3 svc4.example.net. (
-        alpn="bar" port="8004" esniconfig="..." )
+        alpn="bar" port="8004" echo="..." )
```

Comment:
ECHO uses `ECHOConfig` to carry this data -- should we do the same here?

Reply:
Given "echo" has other meetings, that's probably a good plan. "echoconfig" is a hair longer but a more unique string.

Reply:
Done